docs/spec_draft.md

```diff
@@ -1406,6 +1407,61 @@ and produces a `result` tensor. More formally,
 ```
 
 [Back to Ops](#index-of-ops)
+
+## stablehlo.rng
+
+### Semantics
+
+Constructs a `result` tensor of a given shape `shape` with random numbers
+generated using the `rng_distribution` algorithm. If `rng_distribution` $=$
+`UNIFORM`, then the random numbers are generated following the uniform
+distribution over the interval [`a`, `b`). If `rng_distribution` $=$ `NORMAL`,
+then the random numbers are generated following the normal distribution with
+mean = `a` and standard deviation = `b`.
```

Comment:
Also what happens if `b` is negative in the `NORMAL` case?

Reply:
Good point. I think it should be implementation defined as well, since according to the cpp [doc](https://en.cppreference.com/w/cpp/numeric/random/normal_distribution), even though the std is negative, the normal distribution still outputs a number. In the case of negative std, it will "flip" the curve horizontally, but since the distribution curve is symmetrical, I don't know if this is of concern. Very interesting.